src/main/kotlin/util/CPythonCompiler.kt

```diff
@@ -78,7 +81,18 @@ class CPythonCompiler() {
      * Compiles from a string.
      */
     fun compile(data: String): KyCodeObject {
-        val args = listOf("--code", data)
-        return KyCodeObject(executeCompiler(args).code)
+        return if (!isWindows) {
+            val args = listOf("--code", data)
+            KyCodeObject(executeCompiler(args).code)
+        } else {
+            // windows requires a... different approach
+            val tmp = Files.createTempFile("kyc-windows-sucks", ".py")
```

Comment:
Could be simplified with:
```kotlin
val tmp = createTempFile("kyc-windows-sucks", ".py").also {
    it.writeBytes(data.toByteArray(Charset.defaultCharset()))
}
val args = listOf("--path", tmp.absolutePath)
...
tmp.delete()
```
Windows, macOS and Linux also share UTF_8 as default charset, as far as I know.

Reply:
Small addendum: If you don't need a ByteArray, `it.write(data, Charset.defaultCharset()` would also suffice

Reply:
I don't see how that's simpler, this is pretty standard NIO API and it's a line/nest shorter